docs/blog/lion-without-polyfills.md

```diff
@@ -0,0 +1,98 @@
+
+## How does it work?
+
+1. Within lion class files we only import other class files
+2. We define them as `scopedElements` and let the mixin handle the rest
+
+To clarify within lion class files we never import files that run `customElement.define`
+
+```js
+import { LitElement, html } from 'lit';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { LionOptions } from './LionOptions.js';
+
+export class LionListbox extends ScopedElementsMixin(LitElement) {
+  static scopedElements = {
+    'lion-options': LionOptions,
+  };
+
+  render() {
+    return html`
+      <div>
+        <!-- example code - not the real implementation -->
+        <lion-option></lion-option>
+      </div>
+    `;
+  }
+}
+```
+
+## What is wrong with it?
+
+The code above totally makes sense - however we always assumed that a scoped registry will be available.
+Which was somewhat of a valid assumption as all our components are using the `ScopedElementsMixin` and it in turn loads a polyfill for the scoped registry.
+
+We however over time we got multiple feedbacks that lion components "break the app as soon as you load them".
+The reasons is/was that not everyone is always using `ScopedElementsMixin` or in full control of the app (or it's load order).
+
+To quote the release notes of the latest version of `ScopedElementsMixin`:
+
+> ScopedElementsMixin 2.x tried to be as convenient as possible by automatically loading the scoped custom elements registry polyfill.
+> This however led to a fatal error whenever you registered any component before ScopedElementsMixin was used.
+
+And this was the case.
+
+## How do we fix it?
+
+With the latest release of lion we now updated to the latest version of `ScopedElementsMixin` which means lion now works in all apps as long as there is no need for actual scoping.
+
+If you extends lion components and you imperatively create scoped custom elements then you should now use a helper function that will work in scoped and unscoped cases.
+
+```diff
+-  const myButton = this.shadowRoot.createElement('my-button');
++  const myButton = this.createScopedElement('my-button');
+```
+
+## How to get scoping
+
+You need scoping if you want to
+
+- use 2 major versions of a web component (e.g. in an SPA pageA uses 1.x and pageB uses 2.x of color-picker)
+- or you want to use the same tag name with different implementations (use tag color-picker from foo here and from bar here)
+
+If you need scoping you need to install and load the polyfill first thing in your HTML.
+
+```bash
+npm i @webcomponents/scoped-custom-element-registry
+```
+
+It could look something like this:
+
+```html
+<script src="/node_modules/@webcomponents/scoped-custom-element-registry/scoped-custom-element-registry.min.js"></script>
+```
+
+or if you have an SPA you can load it at the top of your app shell code
+
+```js
+import '@webcomponents/scoped-custom-element-registry';
+```
+
```

Comment:
In this scoping section, we should emphasize this is desired behavior and that as soon as scoped registries are supported by the platform, the behavior that you get with loading, will be the default.

Reply:
Then I would expect a section here that refers to the title... (w/o polyfills)
Explain the need for it (perf in small apps) and how it is backw. compatible with enabling the registry (either via browser or polyfill)?